pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -444,21 +444,28 @@ same number of patterns as the number of @racket[val-expr]s.
 
 @examples[#:eval match-eval
 (match* (1 2 3)
- [(_ (? number?) x) (add1 x)])
+  [(_ (? number?) x) (add1 x)])
+
 (match* (15 17)
- [((? number? a) (? number? b))
+  [((? number? a) (? number? b))
   #:when (= (+ a 2) b)
```

Comment:
One more nit: the body should also be further indented.

Reply:
ah, the scribble-mode needs some love